src/module.c

```diff
@@ -6099,6 +6090,38 @@ robj **moduleCreateArgvFromUserFormat(const char *cmdname, const char *fmt, int
     return NULL;
 }
 
+/* Returns an array of robj pointers, by parsing the format specifier "fmt" as described for
+ * the RM_Call(), RM_Replicate() and other module APIs. Populates *argcp with the number of
+ * items (which equals to the length of the allocated argv).
+ *
+ * The integer pointed by 'flags' is populated with flags according
+ * to special modifiers in "fmt".
+ *
+ *     "!" -> REDISMODULE_ARGV_REPLICATE
+ *     "A" -> REDISMODULE_ARGV_NO_AOF
+ *     "R" -> REDISMODULE_ARGV_NO_REPLICAS
+ *     "3" -> REDISMODULE_ARGV_RESP_3
+ *     "0" -> REDISMODULE_ARGV_RESP_AUTO
+ *     "C" -> REDISMODULE_ARGV_RUN_AS_USER
+ *     "M" -> REDISMODULE_ARGV_RESPECT_DENY_OOM
+ *     "K" -> REDISMODULE_ARGV_ALLOW_BLOCK
```

Comment:
i think the comment that describes the flags belongs where it was, i.e. next to the code the handles them (now in the renamed moduleCreateArgvFromUserFormatAux).
this will also mean less changes to the diff

Reply:
The drawback of having this comment before an internal function like moduleCreateArgvFromUserFormat is that it is not visible in the generated documentation. I think it's better to put this in the documentation of RM_Call() and refer to RM_Call() in other commands that use it, such as RM_Replicate()'s documentation.

Reply:
Ignore my previous comment.